### PR TITLE
use description for education instead of institution, faculty

### PR DESCRIPTION
**Pull request by AlexP11223**
we don't really need all these details to be separate and these fields may not be suitable in some cases

### CODE REVIEW

File: app/views/schemas/_resume.json.jbuilder [hunk 1]
```diff
@@ -6,5 +6,5 @@ json.set! :@id, resume_path(resume)
 json.set! :jobTitle, resume.name
 json.set! :familyName, resume.user.last_name
 json.set! :givenName, resume.user.first_name
-json.set! :alumniOf, educations, :institution
+json.set! :alumniOf, educations, :description
```
Comment (mokevnin) [inline app/views/schemas/_resume.json.jbuilder]: Сверился с https://schema.org ?

File: db/migrate/20191004215126_fill_resume_educations_description_for_old_resumes.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class FillResumeEducationsDescriptionForOldResumes < ActiveRecord::Migration[6.0]
+  def change
+    Resume::Education.find_each do |education|
+      education.update(description: [education.faculty, education.institution].select(&:present?).join(', '))
```
Comment (mokevnin) [inline db/migrate/20191004215126_fill_resume_educations_description_for_old_resumes.rb]: Ничо си первый раз на руби. Нормально так.

**Comment by mokevnin** [comment]
Еще хинта для формы не хватает.

File: db/migrate/20191004215126_fill_resume_educations_description_for_old_resumes.rb (hunk 2 above)
Comment (AlexP11223) [reply db/migrate/20191004215126_fill_resume_educations_description_for_old_resumes.rb]: Stackoverflow Driven Development ftw!

**Pull request merged**